Error on Fresh Docker Build

Hi

I'm just checking out this repo for the first time. However, when I attempt to run `npm run docker:build` I get the following error on a freshly checked out project. 

Do you have any thoughts on how to get around this?

```
Step 1/10 : FROM node:14 AS typescript-build
14: Pulling from library/node
76b8ef87096f: Pull complete
2e2bafe8a0f4: Pull complete
b53ce1fd2746: Pull complete
84a8c1bd5887: Pull complete
7a803dc0b40f: Pull complete
b800e94e7303: Pull complete
8e9f42962912: Pull complete
cc1c1f0d8c86: Pull complete
a42c31ab44dd: Pull complete
Digest: sha256:8eb45f4677c813ad08cef8522254640aa6a1800e75a9c213a0a651f6f3564189
Status: Downloaded newer image for node:14
 ---> d6602e31594f
Step 2/10 : ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
 ---> Running in e7cc6468fa76
Removing intermediate container e7cc6468fa76
 ---> 12c02e814c4b
Step 3/10 : COPY /build.sh build.sh
 ---> 85bb5c588b3f
Step 4/10 : COPY .work /adr-gateway
 ---> f588e32a0ea1
Step 5/10 : RUN chmod +x build.sh && ./build.sh
 ---> Running in fc78a30382bc
Installing packages...
npm WARN deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated debug@3.2.6: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated mkdirp@0.5.4: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm WARN deprecated node-pre-gyp@0.11.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated

> sqlite3@5.0.2 install /adr-gateway/node_modules/sqlite3
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download
[sqlite3] Success: "/adr-gateway/node_modules/sqlite3/lib/binding/napi-v3-linux-x64/node_sqlite3.node" is installed via remote

> core-js@2.6.12 postinstall /adr-gateway/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.1 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/cpx/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN adr-gateway@1.3.8 No repository field.
npm WARN adr-gateway@1.3.8 No license field.

added 1042 packages from 1261 contributors and audited 1050 packages in 24.742s

59 packages are looking for funding
  run `npm fund` for details

found 2 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
Typescript version:
Version 3.7.7
Building...

> adr-gateway@1.3.8 build /adr-gateway
> npx rimraf dist && npm run build-templates && npx tsc && npx cpx package.json dist


> adr-gateway@1.3.8 build-templates /adr-gateway
> node ./src/Common/Connectivity/Dependencies.generator.js

/adr-gateway/node_modules/js-yaml/index.js:10
    throw new Error('Function yaml.' + from + ' is removed in js-yaml 4. ' +
    ^

Error: Function yaml.safeLoad is removed in js-yaml 4. Use yaml.load instead, which is now safe by default.
    at Object.safeLoad (/adr-gateway/node_modules/js-yaml/index.js:10:11)
    at Object.<anonymous> (/adr-gateway/src/Common/Connectivity/Dependencies.generator.js:9:25)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! adr-gateway@1.3.8 build-templates: `node ./src/Common/Connectivity/Dependencies.generator.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the adr-gateway@1.3.8 build-templates script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-29T06_19_44_908Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! adr-gateway@1.3.8 build: `npx rimraf dist && npm run build-templates && npx tsc && npx cpx package.json dist`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the adr-gateway@1.3.8 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-29T06_19_44_927Z-debug.log
The command '/bin/sh -c chmod +x build.sh && ./build.sh' returned a non-zero code: 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! adr-gateway@1.3.8 build:docker: `npm run copy-proj && docker build --build-arg HTTP_PROXY --build-arg HTTPS_PROXY examples/deployment/docker -t adr-gateway`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the adr-gateway@1.3.8 build:docker script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2021-04-29T06_19_46_898Z-debug.log
```

Cheers

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "test": "cross-env TEST_SUITE_MOCK_ONLY=1 TEST_SUITE_HEADLESS=1 LOG_LEVEL=silent LOG_SILENT=1 mocha \"dist/Tests/**/*spec.js\"",
     "test-live": "cross-env TEST_SUITE_HEADLESS=1 LOG_LEVEL=silent LOG_SILENT=1 mocha --grep \"Live environments\" \"dist/Tests/**/*spec.js\"",
     "test-coverage": "cross-env TEST_SUITE_MOCK_ONLY=1 TEST_SUITE_HEADLESS=1 LOG_LEVEL=silent LOG_SILENT=1 nyc --reporter text --reporter html --reporter cobertura npm run test",
-    "copy-proj": "npx rimraf examples/deployment/docker/.work && npx cpx \"src/**\" \"examples/deployment/docker/.work/src\" && npx cpx \"{package,tsconfig}.json\" \"examples/deployment/docker/.work\""
+    "copy-proj": "npx rimraf examples/deployment/docker/.work && npx cpx \"src/**\" \"examples/deployment/docker/.work/src\" && npx cpx \"{package,package-lock,tsconfig}.json\" \"examples/deployment/docker/.work\""
   },
   "devDependencies": {
     "@istanbuljs/nyc-config-typescript": "^1.0.1",

